pkgs/tools/graphics/piglit/default.nix

```
@@ -0,0 +1,76 @@
+{ stdenv
+, fetchgit
+, lib
+, cmake
+, freeglut
+, libGL
+, libGLU
+, libglvnd
+, makeWrapper
+, ninja
+, pkg-config
+, python3Packages
+, waffle
+, wayland
+, libX11
+, libXrender
+, libxcb
+, libxkbcommon
+}:
+
+stdenv.mkDerivation rec {
+  pname = "piglit";
+  version = "unstable-2020-10-28";
```

Latest commit is 23 Oct, 2020:
```suggestion
  version = "unstable-2020-10-23";
```